portfolio.html

```diff
@@ -59,5 +60,178 @@ <h6 class="connect">LET'S CONNECT</h6>
         </ul>
       </div>
     </section>
+  </section>
+
+    <!--Portfolio "Works" section -->
+    <!-- Secound page  -->
+
+    <div class="mainContainer">
+      <section class="snapshoot">
+        <img src="img/Snapshoot Portfolio2.png" alt="Snapshoot_portfolio" />
+      </section>
+      <section class="info">
+        <h2 class="name">Multy-Post Stories</h2>
+        <ul class="info_job">
+          <li class="job">CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </section>
+      <section class="decription">
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+      </section>
+      <section class="linkButtons">
+        <button class="link">html</button>
+        <button class="link">css</button>
+        <button class="link">javaScript</button>
+      </section>
+      <div class="seeMore">
+        <button class="more">See Project</button>
+      </div>
+    </div>
+
+    <!-- Third page  -->
+
+    <div class="mainContainer">
+      <section class="snapshoot">
+        <img src="img/Snapshoot Portfolio3.png" alt="Snapshoot_portfolio" />
+      </section>
+      <section class="info">
+        <h2 class="name">Tonic</h2>
+        <ul class="info_job">
+          <li class="job">CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </section>
+      <section class="decription">
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+      </section>
+      <section class="linkButtons">
+        <button class="link">html</button>
+        <button class="link">css</button>
+        <button class="link">javaScript</button>
+      </section>
+      <div class="seeMore">
+        <button class="more">See Project</button>
+      </div>
+    </div>
+
+    <!-- Fourth page  -->
+
+    <div class="mainContainer">
+      <section class="snapshoot">
+        <img src="img/Snapshoot Portfolio4.png" alt="Snapshoot_portfolio" />
+      </section>
+      <section class="info">
+        <h2 class="name">Multy-Post Stories</h2>
+        <ul class="info_job">
+          <li class="job">CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </section>
+      <section class="decription">
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+      </section>
+      <section class="linkButtons">
+        <button class="link">html</button>
+        <button class="link">css</button>
+        <button class="link">javaScript</button>
+      </section>
+      <div class="seeMore">
+        <button class="more">See Project</button>
+      </div>
+    </div>
+    <!-- Portfolio "about me" section -->
+    <!-- ABOUT ME 1 PAGE -->
+    <div class="mainContainerAboutme">
```

Comment:
Kindly use `<section>` for this. This is semantically correct and screen reader are able to read your code easily. Each individual area of your project should be a section on its own